src/app/QuickStarts/quickstarts/settings-quickstart.tsx

```diff
@@ -54,98 +54,95 @@ const SettingsQuickStart: QuickStart = {
+[[APP] currently only supports English. We are planning on adding support for other languages in the future.]{{admonition note}}`,
       },
       {
         title: 'Go to the Connectivity settings tab',
         description: `
-The Connectivity tab allows you to configure the WebSocket connection the browser forms with the Cryostat backend.
+The Connectivity tab enables you to configure the WebSocket connection between the browser and the [APP] backend.
 
-1. Go to the [Connectivity]{{highlight settings-connectivity-tab}} tab.
+1. Click [Connectivity]{{highlight settings-connectivity-tab}}.
 2. Configure the WebSocket Connection Debounce time.
 3. Configure the Auto-Refresh period for content-views.
-[To use the Auto-Refresh feature, you must have the [Auto-Refresh]{{highlight settings-connectivity-tab-auto-refresh}} checkbox enabled.]{{admonition tip}}
+[To use the Auto-Refresh feature, make sure to enable the [Auto-Refresh]{{highlight settings-connectivity-tab-auto-refresh}} checkbox.]{{admonition tip}}
 `,
       },
       {
         title: 'Go to the Notifications & Messages tab',
         description: `
-The Notifications & Messages tab allows you to configure the notifications and deletion warnings that are displayed in the Cryostat UI.
+The Notifications & Messages tab allows you to configure the notifications and deletion warnings that are displayed in the [APP] UI.
 
-1. Go to the [Notifications & Messages]{{highlight settings-notifications&messages-tab}} tab.
-2. Enable or disable notifications from various categories if you choose to.
-3. Enable or disable deletion dialog warnings for various destructive actions.
-[You can also control the maximum number of notifications that can be displayed at once.]{{admonition tip}}
+1. Click [Notifications & Messages]{{highlight settings-notifications&messages-tab}}.
+2. You can enable or disable notifications from various categories. Click \`Show more\` for the full list of notifications.
+3. You are also able to enable or disable deletion dialog warnings for various destructive actions. Click \`Show more\` for the full list of deletion warnings.
+[Control the maximum number of notifications that can be displayed at once with the input control.]{{admonition tip}}
 `,
       },
       {
         title: 'Go to the Dashboard tab',
         description: `
 The Dashboard tab allows you to configure settings for the various Dashboard Cards that you can add to the Dashboard.
 
-New to [APP] 2.3 is the Automated Analysis Recording Configuration card. This card starts a recording and then automatically starts an analysis on the recording. You can configure the recording that is started by this card.
+New to [APP] 2.3 is the **Automated Analysis** card. This card starts a recording and then automatically starts an analysis on the recording. You can configure the recording that is started by this card.
```

Comment:
Start a recording is opt-in. This paragraph might imply that it starts on its own.

Reply:
This was for setting the default configurations right for the custom recording form right?

Reply:
As of now, the settings in both the card and the settings page always set the current AA recording configurations.